src/application-loading-overlay/ApplicationLoadingOverlayProvider.module.scss

```diff
@@ -0,0 +1,5 @@
+:local {
+  .container {
+    height: 100%;
+  }
```

Comment:
linking this comment from https://github.com/cerner/terra-framework/pull/891

https://github.com/cerner/terra-framework/pull/891#discussion_r329210846

Reply:
https://github.com/cerner/terra-framework/pull/891#discussion_r340332603